infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/config/LogReplicationRoutingQueueConfig.java

```diff
@@ -59,9 +59,9 @@ public LogReplicationRoutingQueueConfig(@NonNull LogReplication.LogReplicationSe
         super(session, new HashSet<>(), new HashMap<>(), serverContext);
         this.snapshotSyncStreamTag = SNAPSHOT_SYNC_QUEUE_TAG_SENDER_PREFIX + session.getSinkClusterId();
         this.logEntrySyncStreamTag = LOG_ENTRY_SYNC_QUEUE_TAG_SENDER_PREFIX + session.getSinkClusterId();
-        this.sinkQueueName = TableRegistry.getFullyQualifiedTableName(CORFU_SYSTEM_NAMESPACE, REPLICATED_QUEUE_NAME);
-        this.sinkQueueStreamId = CorfuRuntime.getStreamID(TableRegistry.getFullyQualifiedTableName(CORFU_SYSTEM_NAMESPACE,
-                REPLICATED_QUEUE_NAME));
+        this.sinkQueueName = TableRegistry.getFullyQualifiedTableName(CORFU_SYSTEM_NAMESPACE,
+                REPLICATED_RECV_Q_PREFIX+session.getSourceClusterId());
```

Comment:
currently we have the `DEFAULT_CLIENT_NAME` hard coded in several places and in multiple models as well. So for now I did not add the client name since it will need to come in as part of a bigger change to support multiple client replication.